src/libponyrt/gc/cycle.c

```diff
@@ -139,6 +139,9 @@ DEFINE_HASHMAP(ponyint_perceivedmap, perceivedmap_t, perceived_t,
   perceived_hash, perceived_cmp, ponyint_pool_alloc_size,
   ponyint_pool_free_size, perceived_free);
 
+DECLARE_STACK(ponyint_actorstack, actorstack_t, pony_actor_t);
+DEFINE_STACK(ponyint_actorstack, actorstack_t, pony_actor_t);
```

Comment:
Should we choose a name for this stack instance that makes it more clear that it is only used at termination time?

My first thought when I saw this stack is that we'd be pushing to it at creation time for every actor, which is a much scarier concept in terms of program overhead and coordination than what's actually going on (pushing to it and popping from it at runtime termination).